flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/table/catalog/hive/util/HiveTableUtil.java

```diff
@@ -111,25 +192,20 @@ public static TableSchema createTableSchema(
                         HiveTableUtil.relyConstraint((byte) 0));
         // PK columns cannot be null
         primaryKey.ifPresent(pk -> notNullColumns.addAll(pk.getColumns()));
-        return createTableSchema(
-                fields, hiveTable.getPartitionKeys(), notNullColumns, primaryKey.orElse(null));
-    }
 
-    /** Create a Flink's TableSchema from Hive table's columns and partition keys. */
-    public static TableSchema createTableSchema(
-            List<FieldSchema> cols,
-            List<FieldSchema> partitionKeys,
-            Set<String> notNullColumns,
-            UniqueConstraint primaryKey) {
-        List<FieldSchema> allCols = new ArrayList<>(cols);
-        allCols.addAll(partitionKeys);
+        List<FieldSchema> allCols = new ArrayList<>(fields);
```

Comment:
warning from IDE `Contents of collection 'allCols' are updated, but never queried `

Reply:
Good catch, Removed